### DISCUSSION TITLE
error:type object "config"

### DISCUSSION THREAD

**Discussion by 23akitop** [Lost-MSth/Arcaea-server#188] [Q&A]
运行run.bat出错
没有改秘钥😭![IMG20241123151921.jpg](https://github.com/user-attachments/assets/128b324c-d823-4ae0-8906-18bc78259559)

**Comment by 23akitop** [comment]
> 运行run.bat出错
> 没有改秘钥😭![IMG20241123151921.jpg](https://github.com/user-attachments/assets/128b324c-d823-4ae0-8906-18bc78259559)
> 
> 

糟糕，我图片传不上去了

**Reply by Lost-MSth** [reply]
请问你改了什么，目测表象原因是 `linkplay_server/config.py` 文件有问题。

**Reply by 23akitop** [reply]
> 请问你改了什么，目测表象原因是 `linkplay_server/config.py` 文件有问题。

我之前是这个问题![IMG20241123161219.jpg](https://github.com/user-attachments/assets/6e5f79d2-243a-4e94-bb31-10bb90070827)

然后我就改了好多

**Reply by Lost-MSth** [reply]
呃……你的 `linkplay_server/config.py` 改动了多少啊，默认配置不就能跑吗（）

**Reply by 23akitop** [reply]
> 呃……你的 `linkplay_server/config.py` 改动了多少啊，默认配置不就能跑吗（）

！好的我以为需要改很多…
抱歉麻烦大大了🧎🏻‍♂️
